ToDoApp/ToDoApp/CreateTaskViewModel.swift

```diff
@@ -13,36 +13,65 @@ enum InputErrors: Identifiable, Hashable {
         hashValue
     }
 }
+enum CreateTaskViewMode {
+    case createNewTask
+    case editExistingTask(TaskModel)
+}
 @MainActor
 class CreateTaskViewModel: ObservableObject {
     // MARK: Internal Properties
     @Published var error: InputErrors?
+    @Published var title: String
+    @Published var deadline: Date
+    @Published private(set) var mode: CreateTaskViewMode
```

Comment:
Still not let

Reply:
Sorry I don't quite understand, do you want me to change them to let?

Reply:
yeah, that has to be a constant. mode property is never changed during lifecycle of the CreateTaskViewModel